src/components/ActionPlanForm.tsx

```diff
@@ -1,1213 +1,2231 @@
+                                                                            <li>
+                                                                                <Typography
+                                                                                    variant="h6"
+                                                                                    style={{
+                                                                                        fontFamily:
+                                                                                            'Arimo',
+                                                                                    }}
+                                                                                >
+                                                                                    Check
+                                                                                    that
+                                                                                    your
+                                                                                    goal
+                                                                                    can
+                                                                                    be
+                                                                                    measured
+                                                                                    by
+                                                                                    <br />
+                                                                                    the
+                                                                                    CHALK
+                                                                                    observation
+                                                                                    tool
+                                                                                    (e.g.
+                                                                                    Reduce
+                                                                                    <br />
+                                                                                    center
+                                                                                    time
+                                                                                    clean-up
+                                                                                    transition
+                                                                                    to
+                                                                                    7
+                                                                                    <br />
+                                                                                    minutes).
```

Comment:
Now that's a good counterargument for not having Prettier's indent size set to 4 (or keeping things within more components :smile:)